lib/pngdriver/pngdriver.h

```diff
@@ -54,17 +54,13 @@ extern void read_image(void);
 extern void read_ppm(void);
 extern void read_pgm(void);
 extern void read_bmp(void);
-#ifdef HAVE_PNG_H
 extern void read_png(void);
-#endif
 
 extern void write_image(void);
 extern void write_ppm(void);
 extern void write_pgm(void);
 extern void write_bmp(void);
-#ifdef HAVE_PNG_H
 extern void write_png(void);
-#endif
```

Comment:
Is this HAVE_PNG_H removed intentionally?

Reply:
I'm not sure... Let me check

Reply:
This PR now removes it right now, so you have to add it here if I understand correctly.

Reply:
added back with 154c187